Library/Formula/viennacl.rb

```diff
@@ -0,0 +1,20 @@
+class Viennacl < Formula
+  desc "ViennaCL is a linear algebra library leveraging parallel computation."
+  homepage "http://viennacl.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/viennacl/1.6.x/ViennaCL-1.6.2.tar.gz"
+  sha256 "e46e1960b0bd87619bf02a884611ca087ac3c7378b35d5be3d8185f4fe669805"
+  head "https://github.com/viennacl/viennacl-dev.git"
+
+  depends_on "cmake" => :build
+  depends_on :macos => :snow_leopard
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+    prefix.install "#{buildpath}/examples/benchmarks/dense_blas-bench-cpu" => "test"
```

Comment:
The old test seemed highly dependent on OpenCL configuration. Switched to a CPU-only test.